onadata/apps/api/tests/viewsets/test_data_viewset.py

```diff
@@ -1528,6 +1528,32 @@ def test_delete_submission_by_owner(self):
         response = view(request, pk=formid)
         self.assertEqual(len(response.data), 3)
 
+    def test_deleted_xform_instance_captures_user(self):
```

Comment:
Can we add the `deleted_by` check in the existing `test_delete_submission` function to minimize duplication of tests? https://github.com/onaio/onadata/blob/b4a8070af20ed0b797d7ee7556be36f43bf846e9/onadata/apps/api/tests/viewsets/test_data_viewset.py#L1352